### ISSUE TITLE
Issue with minZoom and maxZoom configuration

### ISSUE THREAD

**Issue by Toflar** [geoadmin/mf-chsdi3#3486]
Hi everyone,

First issue report so sorry if this is the wrong place.

I'm trying to pass a minimum and maximum zoom level to the view as documented here: https://openlayers.org/en/latest/examples/zoom-constrained.html
This is not working, as you can see in this example: https://jsfiddle.net/9txjfec4/

I'm not sure but I think we hit https://github.com/openlayers/openlayers/pull/7521 here. 
From what I could see this was released with 4.6.0.

Are there any plans to provide an integration with the latest 6.x version? Or how am I supposed to benefit from the latest bugfixes in OpenLayers?

Thanks in advance :)

**Comment by procrastinatio** [comment]
Hello,
Yes, the geoadmin JS API is very dated now (as you mentioned, based on OL 4.4.2) and there is no plan to upgrade.

I advise you to use pure OL (or any framework). The easiest way is to use the WMTS GetCapabilities document, but it's a bit overkill for only one layer. Alternatively, configuring a WMTS layer is not too complicated (e.g. https://codepen.io/geoadmin/pen/GZKEam, but this example is also very dated), even if you use LV95 (EPSG:2056). If you are using the ubiquitous EPSG:3857, it's even easier.

For any discussion, you may use
https://groups.google.com/forum/#!forum/geoadmin-api

Marc

**Comment by Toflar** [comment]
Hi Marc

We’ve managed to get things working but it was a bit of a hassle.
Not every layer can be requested using WMTS, the attribution has to be added manually etc.
So I ended up checking out the source code of the the version 4.4.2 custom OL build (had to read through the Python code to find out I could add `-debug` to the URL) to find out how `ga.layer.create()` works to mimic the behaviour.
The main issue here - I guess - is that the current documentation is very misleading.
If the custom OL build is not going to be supported anymore then it shouldn’t be endorsed the way it currently is on https://api3.geo.admin.ch.
Everything, from setup instructions to examples is based on something that apparently isn’t going to be continued anymore.
That's unfortunate and it would be great if it showed raw OL usage instead :)

Anyway, thanks for taking the time to reply!
Closing.
